### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by piqusy**
Homework 1

- [x] HTML
- [x] CSS
- [x] JavaScript

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+<p>Morpheus explains that in reality, intelligent machines have enslaved humanity, using their bodies as an energy source while keeping their minds imprisoned in the Matrix. Neo learns he is "The One," prophesied to liberate humanity from the machines. Morpheus and his crew, including the skilled fighter Trinity (Carrie-Anne Moss), train Neo to manipulate the Matrix's code, giving him superhuman abilities within the simulation.</p>
+
+<p>The group battles against the Matrix's sentient security programs, known as Agents, led by the relentless Agent Smith (Hugo Weaving). As Neo's powers grow, he confronts his destiny, culminating in a climactic showdown with Agent Smith. Neo ultimately embraces his role as humanity's savior, heralding the beginning of the fight to free humanity from the Matrix's grip. The film ends with Neo's promise to expose the truth and lead the resistance against the machines.</p></div>
+        <img class="movie__poster" src="./../assets/matrix.png" alt="IMDb Logo" />
```
Comment (darrac) [inline static/index.html:26]: Sto dobivas s ovim da ti je img zadnji i onda settas order kroz css? Zasto ne samo staviti img tag gore? Sto ne kuzim?

**Review by darrac — COMMENTED** [review]
Nemam sto posebno dodati. Ovo sto si napravio je iznad ocekivanja za ovu zadacu u smislu koristenja varijabli, inline paddinga, semantickih elemenata, ispravnih html atributa, samo sto ti fali 60% zadace.

File: static/index.html (hunk 1 above)
Comment (piqusy) [reply static/index.html:26]: Iz semantičke i accesibility perspektive bolje je imati heading kao prvi element

Comment (darrac) [reply static/index.html:26]: Ne kazem da nisi u pravu, ali hejtam to.

Comment (piqusy) [reply static/index.html:26]: Don't hate the player, hate the game 🤣

File: static/index.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      </header>
+
+      <section>
+        <h2 class="site__title">Movie Reviews</h2>
+
+        <form class="review__form">
```
Comment (darrac) [inline static/index.html:32]: You did not close the form tag

File: static/index.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Movies APP</title>
+    <link href="./../static/style.css" rel="stylesheet">
+  </head>
+  <body>
+    <main>
+      <h1 class="site__title">Movies APP</h1>
+
+      <header class="movie__wrapper">
+        <h2 class="movie__title">The Matrix</h1>
```
Comment (darrac) [inline static/index.html]: This heading here is not sure if it's an h1 or an h2

File: static/index.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+            <label class="rating__label">
+              <span class="rating__star" aria-hidden="true"></span>
+              <div class="rating__number">5</div>
+              <input class="rating__radio" type="radio" id="rating-5" name="rating" value="5" />
+            </label>
+
+            <label class="rating__label">
+              <span class="rating__star" aria-hidden="true"></span>
+              <div class="rating__number">4</div>
```
Comment (darrac) [inline static/index.html:45]: What are these divs used for?

File: static/index.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+  return [];
+}
+
+function saveReviews(reviews) {
+  localStorage.setItem('reviews', JSON.stringify(reviews));
+}
+
+function addReview() {
+  const reviewInput = document.getElementById('movie-review');
```
Comment (darrac) [inline static/index.js:76]: This does not prevent the default behaviour so you get a validation error after adding the review.

File: static/index.html (hunk 4 above)
Comment (piqusy) [reply static/index.html:45]: `rating__number`? For accessibility

Comment (piqusy) [reply static/index.html:45]: But if the radio input visibility is handled differently then could be removed

Comment (darrac) [reply static/index.html:45]: Wouldn't a label with a correct for attribute be better?

Comment (piqusy) [reply static/index.html:45]: That does nothing accessibility wise.
If you are referring the `label` relationship to the `input radio` field, the `radio` is inside the `label` making the `for` attribute redundant.

Comment (darrac) [reply static/index.html:45]: That is what I was referring to. I missed that it is wrapped, it all makes sense now :)

File: static/index.js (hunk 5 above)
Comment (piqusy) [reply static/index.js:76]: I'm having issues with the the usual `event.preventDefault()` here.
Can I get some assistance pls?

Comment (piqusy) [reply static/index.js:76]: NVM - I may have found the issue

**Review by darrac — APPROVED** [review]

**Review by darrac — APPROVED** [review]

**Pull request merged**